app.go

```diff
@@ -131,6 +131,27 @@ func (io invokeOption) String() string {
 	return fmt.Sprintf("fx.Invoke(%s)", strings.Join(items, ", "))
 }
 
+// Error registers any number of errors with the application to short-circuit
+// startup. If more than one error is given, the errors are combined into a
+// single error, where each of the error messages are delimited by a newline.
```

Comment:
Don't mind taking varargs here, but let's not document the details of the error formatting - that's not part of the API contract.

This option is a little less self-explanatory than the others, so a small testable example would be nice.

Reply:
:+1: